### DISCUSSION TITLE
Cannot create a folder without an error message via a CIFS / Samba share in a mergerfs pool with Finder or Explorer. Commandline is OK.

### DISCUSSION THREAD

**Discussion by minenko** [trapexit/mergerfs#1341] [Q&A]
I use mergerfs since years on my Linux-based archive server where I read/write files via rsync. Now, I wanted to use mergerfs on a Synology NAS. I've set it up in similar way as on this archive server started with an older version which comes via Entware. The use case is to expert the pool as a folder via "SMB" share. It generally works, but if I try to create a folder via Explorer in Windows or in Finder in MacOS, a folder with a default name is created fist - "New folder" in Windows and "untitled folder" on macOS, and try to five it the right name in place, both Windows and macOS, report an error first, but than the folder is created with the right name. If works ok, if I do not set the right name, but just leave the default name. I tried a few options in mergerfs and upgraded to the newest version 2.40.2, but the effect persists. Any ideas how to avoid it?  Thanks!

**Comment by trapexit** [comment]
Without knowing the error and seeing exactly what the server is doing to mergerfs or having your settings I can't help.

https://github.com/trapexit/mergerfs#support

**Reply by minenko** [reply]
Thanks! I understand! I thought, I got a known issue. Let me collect these information.
